### PR TITLE
Update #workspaceRoot variables reference

### PR DESCRIPTION
**Pull request by torn4dom4n**
Detail: https://code.visualstudio.com/docs/editor/variables-reference#_why-isnt-workspaceroot-documented

### CODE REVIEW

**Comment by ramya-rao-a** [comment]
Can you also update the `package.json` and README files? There are a few references to `$workspaceRoot` there as well.

**Comment by ramya-rao-a** [comment]
Also, dont forget to register for the [Microsoft Hacktoberfest](https://open.microsoft.com/2018/09/18/hacktoberfest-2018-microsoft/) :)

File: src/util.ts [hunk 1]
```diff
@@ -350,16 +350,16 @@ function resolveToolsGopath(): string {
...
 		return resolvePath(toolsGopathForWorkspace);
 	}
 
-	// In case of multi-root, resolve ~ and ignore ${workspaceRoot}
+	// In case of multi-root, resolve ~ and ${workspaceFolder}
 	if (toolsGopathForWorkspace.startsWith('~')) {
 		toolsGopathForWorkspace = path.join(os.homedir(), toolsGopathForWorkspace.substr(1));
 	}
-	if (toolsGopathForWorkspace && toolsGopathForWorkspace.trim() && !/\${workspaceRoot}/.test(toolsGopathForWorkspace)) {
+	if (toolsGopathForWorkspace && toolsGopathForWorkspace.trim() && !/\${workspaceFolder}/.test(toolsGopathForWorkspace)) {
```
Comment (ramya-rao-a) [inline src/util.ts]: we should support both `$workspaceRoot` and `$workspaceFolder` here. Not everyone would have updated their settings.

File: src/util.ts [hunk 2]
```diff
@@ -502,21 +502,21 @@ export function timeout(millis): Promise<void> {
 }
 
 /**
- * Exapnds ~ to homedir in non-Windows platform and resolves ${workspaceRoot}
+ * Exapnds ~ to homedir in non-Windows platform and resolves ${workspaceFolder}
  */
-export function resolvePath(inputPath: string, workspaceRoot?: string): string {
+export function resolvePath(inputPath: string, workspaceFolder?: string): string {
```
Comment (ramya-rao-a) [inline src/util.ts]: The `resolvePath` function was already supporting both `$workspaceRoot` and `$workspaceFolder`. We want to retain that. So, please revert the changes done to `resolvePath` function

File: src/util.ts (hunk 1 above)
Comment (ramya-rao-a) [inline src/util.ts]: we should support both `$workspaceRoot` and `$workspaceFolder` here. Not everyone would have updated their settings.

File: src/util.ts (hunk 2 above)
Comment (ramya-rao-a) [inline src/util.ts]: The `resolvePath` function was already supporting both `$workspaceRoot` and `$workspaceFolder`. We want to retain that. So, please revert the changes done to `resolvePath` function

**Comment by torn4dom4n** [comment]
Sorry @ramya-rao-a I didn't read it. I see you did everything 😆.

**Comment by ramya-rao-a** [comment]
No problem :) I was going through all the PRs and wrapping them up to be ready for the next update
